src/jit/codegenarm.cpp

```diff
@@ -19,6 +19,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 #ifdef _TARGET_ARM_
 #include "codegen.h"
+#include "sideeffects.h"
```

Comment:
Personally, I feel that headers should not include their dependencies, as it makes discovering which headers are actually necessary for a particular source file more difficult. However, I think I've been outvoted on this one :)